### ISSUE TITLE
Activity feed feature: screenshots and discussion [code almost complete]

### ISSUE THREAD

**Issue by jonkerz** [calacademy-research/antcat#131]
**TL;DR**: We should list most catalog changes/events in an activity feed like this `"Quintus Batiatus edited the reference Joffre (2004) 2 hours ago"`.

**Problem 0**: We need this, but do not have it.

**Problem 1**: Committing a change that you have no idea how it affects the database and you don't even know if it can be reversed, is scary. It's scary for the user, who may unknowingly mess up the catalog, and it's scary for the team, who may not be comfortable with allowing new editors to edit the catalog because it's too hard to track changes. Hard to edit and track changes = less contributors, and less edits from each contributor.

**Problem 2**: Our current catalog change feed only includes edits made to taxa (and not even all types of changes). That code cannot really be expanded to include for example new events for `"Zxy edited the reference ..."`.

**More**:
* Likely to be merged with the current changes feature in the future, in some way or another.
* Note about why some actions/events are hard to add to the feed and/or make their version history browsable: some saves triggers multiple saves to unchanged/empty fields; for example saving a reference without changing anything results in, say, about 5 saves -- these are all added as new versions in the database despite that they're really not, and it will take a lot of effort to cleanup all related code.

#### TODO
- [ ] Check if #36, "Changes is switching editors", is still an issue

**Comment by jonkerz** [comment]
#### Screenshot from dev:
![01_feed](https://cloud.githubusercontent.com/assets/1513381/14361520/8413188e-fcfb-11e5-82e6-4ecfee9e25f7.png)

TODO list implemented events/actions.

**Comment by jonkerz** [comment]
#### Beta/WIP/dev
**Bonus screenshot of a version/history browser**
This is more related to a new feature that has not yet been implemented or even prototyped, but I have implemented a very primitive history browser for taxon history items. The feed currently links this page from new/edited history items listed in the feed (not shown in the above screenshot). Making it work for taxa objects --which is what we _really_ want to be able to [diff](https://en.wikipedia.org/wiki/Data_comparison) -- is harder, but it's on the roadmap.
![primitive_version_browser](https://cloud.githubusercontent.com/assets/1513381/14404532/350dd9c0-fe79-11e5-9e4c-7338d333ad88.png)
